src/Loader.php

```diff
@@ -157,6 +160,32 @@ public static function add_blocks(): void
         new Blocks\GuestBook();//NOSONAR
     }
 
+      /**
+     * Load editorIndex file for beta blocks.
+     */
+    public static function add_beta_blocks(): void
+    {
+        if (!BetaBlocks::is_active() && !AllowAllBlocks::is_active()) {
```

Comment:
I think it makes more sense to check for Planet4Blocks flag instead of AllBlocks (in addition to Beta)

Reply:
We need the `All Blocks` Feature enabled so we have access to the `Query Loop Block` which the `Posts List block` extends, if not, it does not show up in the editor.

Now that I think about it, is it redundant to have the check for `AllBlocks` here, because it still won't show up if that is not enabled but `Beta Blocks` is 🤔 ?